desktop-src/SecAuthZ/app-isolation-overview.md

```diff
@@ -8,7 +8,7 @@ ms.date: 10/01/2024
 
 # Win32 app isolation overview
 
-The Win32 app isolation security feature for Windows provides a sandbox environment that can integrated into Win32 apps, providing an additional layer of security. This enhancement requires little to no change to your code.
+The Win32 app isolation security feature for Windows provides a sandbox environment that can integrated into Win32 apps, providing an additional layer of security. This enhancement requires little to no change to your code. If you have a feature request or wish to report a buf, please, file an issue in our GitHub [repo](https://github.com/microsoft/win32-app-isolation/issues).
```

Comment:
I believe it's a `bug`, not a `buf`. Also I'd like to break this line. In `md` file, a new line does not really create a new line

```md
Some sentence.
Another sentence.
```

Only
```md
Some paragraph.

Another paragraph.
```

Will rendered as a new paragraph.
Renders to 

```
Some sentence. Another sentence.
```

Reply:
Also not sure if the `please` in the middle matches the overall tone of a documentation. I think we should be more objective and less personal.

Reply:
@tiangao-ms I followed your instructions, good sir. Are we good?